app/Domain/Tickets/Services/Tickets.php

```diff
@@ -1232,14 +1232,16 @@ public function quickAddMilestone($params): array|bool|int
          */
         public function addTicket($values)
         {
+            $userId = $values['userId'] ?? session("userdata.id");
```

Comment:
consider leaving a comment here to explain that zapier doesn't have user data sessions available

Reply:
you are correct sessions do work ✅